sintr_worker/test/basic_info_test.dart

```diff
@@ -57,6 +57,37 @@ Data :: ~1427814186420:Noti:{"event"::"server.connected","params"::{"version"::"
     var result = await runMap(data);
     expect(result, hasLength(0));
   });
+
+  test('reduce', () async {
+    StreamController<String> controller = new StreamController();
+    var uuid1 = '1427375441782463424515';
```

Comment:
Could you add a bit of whitespace (a few empty lines) to break this up between setup, execution and verification/?